presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -559,6 +561,7 @@ For the Hive connector, a table scan can be delayed for a configured amount of
 time until the collection of dynamic filters by using the configuration property
 ``hive.dynamic-filtering-probe-blocking-timeout`` in the catalog file or the catalog
 session property ``<hive-catalog>.dynamic_filtering_probe_blocking_timeout``.
+Delaying table scans is not supported for dynamic filters on bucketed columns.
```

Comment:
That's not true. If join is not source stage, it should work. I would skip this sentence as it might confuse readers.

Reply:
removed